.devcontainer/Dockerfile

```
@@ -1,4 +1,15 @@
-FROM mcr.microsoft.com/devcontainers/go:1.22-bookworm
+ARG GO_VERSION=1.24.1
+
+# ビルド変数で指定したバージョンのgolangを取得
+FROM alpine:latest AS latest-go-fetcher
+ARG GO_VERSION
+ENV GO_VERSION=${GO_VERSION}
+
+RUN apk add --no-cache curl tar
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)